core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -592,7 +597,7 @@ class SpendReportingService(
               }
             } else {
               val query = getAllUserWorkspaceQuery(tableNameForQuery, billingProjectsByAccount, pageSize, offset)
-              val queryJob = setUpAllUserWorkspaceQuery(query, start, end)
+              val queryJob = setUpAllUserWorkspaceQuery(tableNameForQuery, query, start, end)
```

Comment:
Since we execute one request to BigQuery for each spend report table, using the spend report table's name as a unique tag to disambiguate queries feels safe.